blocks/columns/default.css

```diff
@@ -57,7 +57,7 @@
   display: block;
 }
 
-main .columns span.icon > img {
+main .columns:not(.xxxl-icons) span.icon > img {
```

Comment:
nit: I would argue it makes sense to add all the icon size classes here for consistency ;)